tests/framework/helpers/HtmlTest.php

```diff
@@ -671,11 +670,11 @@ public function testRemoveCssClass()
     {
         $options = ['class' => 'test test2 test3'];
         Html::removeCssClass($options, 'test2');
-        $this->assertEquals(['class' => 'test test3'], $options);
+        $this->assertEquals(['class' => ['test', 2 => 'test3']], $options);
```

Comment:
`$options['class']` can be an `array` since all the methods that deal with HTML tags call `renderTagAttributes()` which can accept arrays for this key.

```php
$options['class'] = 'hello';
Html::addCssClass($options, 'world');
echo Html::tag('div', '', $options); // will echo '<div class="hello world"></div>'
```

This behavior keeps compatible with 2.0.x using this patch.